src/actors/objects/mod.rs

```diff
@@ -96,51 +94,6 @@ struct FetchFileMetaRequest {
 #[derive(Clone, Debug)]
 struct FetchFileDataRequest(FetchFileMetaRequest);
 
-#[derive(Debug, Clone)]
-enum FileId {
```

Comment:
Move things type (and renaming it) was the main driver for this thing, because it needs to be accessible from the download service without being unreasonable.